ext/hal/ti/simplelink/source/ti/devices/cc13x2_cc26x2/startup_files/ccfg.c

```diff
@@ -495,15 +498,8 @@
 // Customer Configuration Area in Lock Page
 //
 //*****************************************************************************
-#if defined(__IAR_SYSTEMS_ICC__)
```

Comment:
I'd ask we keep the IAR & TI code here, and just handle the `#else` case

Reply:
OK. Fixed.